### ISSUE TITLE
Compatibility with Mi Notebook Pro version 2019 I5 MX250

### ISSUE THREAD

**Issue by xamichoch** [daliansky/XiaoMi-Pro-Hackintosh#380]
Hey guys,

i read that the guide here is just for version 2017 and 2018, which unfortunatelly seem
impossible to find and buy now.
So i was wondering to get a new version 2019, which has some little hardware improvements but i fear that it won't fit with the current guide.
Anyone has some suggestion or experience?

Regards

**Comment by Menchen** [comment]
#367

**Comment by xamichoch** [comment]
> #367

I guess that the post you linked is just for enhanced version which features an 10th generation cpu.
I wonder if the 2019 version which i'm asking for, is connected to the 2017 and 2018 versions which it shares with the cpu or with the enhancend version with the same gpu MX250 (even if we won't use it).

**Comment by stevezhengshiqi** [comment]
If it's not a PM981, then it's OK to use this repository.

**Comment by Menchen** [comment]
If you got PM981 you could try https://github.com/acidanthera/NVMeFix/blob/master/README.md
Not tested, but very good source.

**Comment by stevezhengshiqi** [comment]
@Menchen According to dozens of people's tests, NVMeFix v1.0.2 will make PM981 worse on macOS. People with PM981 have to remove NVMeFix when they use release from this repository. More information can be found at https://github.com/acidanthera/bugtracker/issues/841

**Comment by xamichoch** [comment]
So, if a get it right, the problem lies in the specific model of nvme, which isn't compatible.
The solution could be to change model of nvme?

**Comment by renyichan** [comment]
I used this EFI and patched pm981. At present, both clover and OpenCORE have successfully started the system. However, I didn't find that OpenCORE is better than clover. Nvmefix I tested that there is no problem in putting it together at present, but it has a function to solve the problem of pm981, which can't be upgraded online, so when upgrading, nvmefix and pm981 patches can't coexist. However, no other problems were found during use. But I still want to take a QQ group to see how they feel
